storage/stiface/stiface_test.go

```diff
@@ -0,0 +1,258 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stiface
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"golang.org/x/net/context"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+func TestIntegration_Public(t *testing.T) {
```

Comment:
What does `_Public` and `_Private` signify?

Reply:
Bad names. Public means you can access the bucket without creds. 

Come to think of it, there's no real point to the Public test. I'll just remove it completely and remove the "_Private" from the other one, if that's OK with you.

Reply:
Ahh ok. Yeah, that SGTM.